File: Conferences.Domain/Entities/Conference.cs
namespace Conferences.Domain.Entities;
public class Conference
{
    public int? Id { get; set; }

    public string Name { get; set; } = string.Empty;

    public DateTime StartTime { get; set; }

    public DateTime EndTime { get; set; }

    public string Location { get; set; } = string.Empty;

    public string Theme { get; set; } = string.Empty;

    public int MaxNumberOfPeople { get; set; }

    public string Language { get; set; } = string.Empty;

    public List<Section>? Sections { get; set; } = new List<Section>();
}

File: Conferences/ClientApp/src/app/models/Conference.ts
import { Section } from "./Section"

export class Conference {
  id: number | null = null
  name: string = ''
  startTime: Date = new Date(Date.now())
  endTime: Date = new Date(Date.now())
  location: string = ''
  theme: string = ''
  maxNumberOfPeople: number = 0
  language: string = ''
  sections: Section[] | null = null
}
